expression/builtin_encryption.go

```diff
@@ -362,9 +363,61 @@ type builtinCreateDigestSig struct {
 	baseBuiltinFunc
 }
 
+// Supported digest type name
+const (
+	SHA224NAME string = "SHA224"
```

Comment:
The Golang standard libs use MixedCaps for const name. So it is better to rename it to SHA224Name.